docs/src/app/privacy-policy/page.tsx

```diff
@@ -1,15 +1,13 @@
-import Navigation from '../../components/navigation';
 import PrivacyPolicy from '../../components/privacy-policy';
 import Footer from '../../components/footer';
 
 export default function PrivacyPolicyPage() {
   return (
-    <div className="min-h-screen">
-      <Navigation />
+    <>
       <main>
```

Comment:
Do we want to move the `<main>` up to `/docs/layout.tsx` so that we don't have to keep track of what has a `main` element?

Reply:
Yeah it's not optimal.. I went down this route because the landing pages `/` + `/thank-you` etc and `/docs` have different layouts and different looking <footer>s, and this way the <nav> and <footer> on /docs-pages are not inside the <main> element. So I struggled to figure out a better way..

But maybe nesting <nav> inside <main> on /docs is fine, since it's a "sub menu" anyways (same goes for the TOC which I was wondering if it should be an <aside>)..

So then I'd only have to solve for the different footers 🤔 which the only solution I can come up with is: 
In the <Footer /> component, do the same `pathname?.startsWith('/docs');` to not render any markup from the `/docs/src/app/layout.tsx` if it's `true`.. Hmmm what you reckon?